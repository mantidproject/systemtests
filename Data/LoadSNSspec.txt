#F norm: /SNS/REF_M/IPTS-1677/37/5394/NeXus/REF_M_5394.nxs
#F data: /SNS/REF_M/IPTS-1677/37/5389/NeXus/REF_M_5389.nxs
#E 1281451285.74
#D 2010-08-10T10:41:25.736946-04:00
#C norm Run Number: 5394
#C data Run Number: 5389
#C data Title: _No2
#C norm Title: _No2Rep0
#C norm Notes: NONE
#C data Notes: NONE
#C data Sample: Name; STO/LMO bilayer on LSAT | Nature; 1
#C norm Proton Charge: (1414559039490.0, 'picoCoulomb')
#C data Proton Charge: (5437422829570.0, 'picoCoulomb')
#C data theta: (0.016452999999999999, 0.0, 'radians')
#C HLR Version - 1.7.12
#C DOM Version - 1.7.2
#C SCL Version - 1.4.1

#S 23 Spectrum ID ('bank1', (0, 127))
#N 3
#L lambda_T(Angstroms)  Intensity(Counts/A)  Sigma(Counts/A)
143.800898828   0.0   0.0
148.294676917   7.90340948874e-05   7.90341941899e-05
152.788455005   3.22318135969e-05   3.22318540946e-05
157.282233094   0.0   0.0
161.776011182   2.63040177974e-05   2.63040377752e-05
166.26978927   6.7180656839e-05   5.48423650766e-05
170.763567359   5.48951106922e-05   5.48951714491e-05
175.257345447   0.0   0.0
179.751123535   5.25098615359e-05   5.25099217973e-05
184.244901624   1.09732239929e-05   1.0973236586e-05
188.738679712   0.0   0.0
193.232457801   0.0   0.0
197.726235889   0.0   0.0
202.220013977   0.0   0.0
206.713792066   1.39097669773e-05   8.03084255786e-06
211.207570154   0.000173282599639   0.000100045189684
215.701348243   1.11093996289e-05   6.41404269944e-06
220.195126331   0.000143963826194   8.31177370796e-05
224.688904419   0.00010472265846   5.4923539104e-05
229.182682508   0.000160665301643   7.18602338421e-05
233.676460596   0.000212685884702   8.91838123196e-05
238.170238685   0.000103939200959   7.34962294008e-05
242.664016773   4.83656659527e-06   4.83657501485e-06
247.157794861   6.95039729862e-05   6.950409398e-05
251.65157295   7.10506620771e-05   6.60569695943e-05
256.145351038   1.5600311964e-05   1.56003332319e-05
260.639129126   0.0   0.0
265.132907215   4.13981814185e-05   4.13982120506e-05
269.626685303   5.48830114971e-05   5.48830521073e-05
274.120463392   3.44440621605e-05   2.43557139876e-05
278.61424148   0.000178317185075   0.000113147142377
283.108019568   0.000320198662676   0.000160100619561
287.601797657   0.000101149545156   7.50370484905e-05
292.095575745   1.85443018411e-05   1.85443293824e-05
296.589353834   0.000104888382565   7.4167392519e-05
301.083131922   0.000337884494544   0.00011947757673
305.57691001   0.000345650412864   0.000141111847109
310.070688099   1.96244980738e-05   1.96245399441e-05
314.564466187   9.69796060573e-05   9.697981297e-05
318.465643453

#S 24 Spectrum ID ('bank1', (1, 127))
#N 3
#L lambda_T(Angstroms)  Intensity(Counts/A)  Sigma(Counts/A)
143.800898828   0.0   0.0
148.294676917   0.0   0.0
152.788455005   0.000163804795806   9.45729518923e-05
157.282233094   0.000116610787385   6.73254182289e-05
161.776011182   0.0   0.0
166.26978927   1.42117323455e-05   1.42117480748e-05
170.763567359   6.00122697235e-05   5.51331591605e-05
175.257345447   5.62588574806e-05   5.62589370196e-05
179.751123535   5.55155592215e-05   5.25958756109e-05
184.244901624   5.89677764687e-05   4.9233043352e-05
188.738679712   6.76084427023e-05   5.1389542041e-05
193.232457801   3.48657417224e-05   3.48657629558e-05
197.726235889   6.30786095971e-05   3.64185245065e-05
202.220013977   0.000157838620597   7.85800541862e-05
206.713792066   0.000108261394675   7.34205071604e-05
211.207570154   5.77608665462e-05   5.77609500961e-05
215.701348243   3.70313320963e-06   3.70313856613e-06
220.195126331   0.0   0.0
224.688904419   4.0538898315e-05   4.0538920433e-05
229.182682508   9.59081416326e-05   5.53817143225e-05
233.676460596   8.26497847639e-05   5.84422884937e-05
238.170238685   0.0   0.0
242.664016773   0.0   0.0
247.157794861   0.0   0.0
251.65157295   0.0   0.0
256.145351038   0.00011584189527   8.1912732589e-05
260.639129126   5.39381019516e-05   3.81400641904e-05
265.132907215   8.2796362837e-05   5.85459562604e-05
269.626685303   0.000140520744769   8.34875000884e-05
274.120463392   7.04772309347e-05   7.04773064568e-05
278.61424148   4.71505865225e-06   4.71506813708e-06
283.108019568   8.00496656689e-05   8.00498266972e-05
287.601797657   6.84280074228e-06   6.8428145073e-06
292.095575745   0.0   0.0
296.589353834   0.0   0.0
301.083131922   4.18252084038e-05   4.18252408575e-05
305.57691001   8.8657647535e-05   6.544303671e-05
310.070688099   0.000133266304988   8.17904693099e-05
314.564466187   0.000290938818172   0.000167974680137
318.465643453

#S 25 Spectrum ID ('bank1', (2, 127))
#N 3
#L lambda_T(Angstroms)  Intensity(Counts/A)  Sigma(Counts/A)
143.800898828   7.74866630574e-06   7.74870222905e-06
148.294676917   7.90340948874e-05   7.90341941899e-05
152.788455005   3.22318135969e-05   3.22318540946e-05
157.282233094   0.0   0.0
161.776011182   7.89120533922e-05   4.55599990761e-05
166.26978927   0.00015890677348   9.17450774848e-05
170.763567359   5.11715903127e-06   5.11716626593e-06
175.257345447   5.62588574806e-05   5.62589370196e-05
179.751123535   5.55155592215e-05   5.25958756109e-05
184.244901624   0.000106962328944   6.87559476162e-05
188.738679712   8.80816679672e-05   5.53176276817e-05
193.232457801   3.48657417224e-05   3.48657629558e-05
197.726235889   2.1026203199e-05   2.10262171085e-05
202.220013977   5.68314963161e-05   4.59528021991e-05
206.713792066   5.18124028412e-05   5.18124575473e-05
211.207570154   0.0   0.0
215.701348243   0.0   0.0
220.195126331   0.0   0.0
224.688904419   0.0   0.0
229.182682508   3.23785800052e-05   3.2378598073e-05
233.676460596   4.1324892382e-05   4.1324915442e-05
238.170238685   0.0   0.0
242.664016773   0.0   0.0
247.157794861   0.0   0.0
251.65157295   0.0   0.0
256.145351038   5.79209476348e-05   5.7920998157e-05
260.639129126   7.72300452023e-05   5.70394771082e-05
265.132907215   8.15121026229e-05   5.76449548481e-05
269.626685303   8.5637733272e-05   6.29126723894e-05
274.120463392   7.04772309347e-05   7.04773064568e-05
278.61424148   4.71505865225e-06   4.71506813708e-06
283.108019568   8.00496656689e-05   8.00498266972e-05
287.601797657   6.84280074228e-06   6.8428145073e-06
292.095575745   0.0   0.0
296.589353834   0.000157332573847   9.08361984868e-05
301.083131922   0.000130399866181   7.52865591497e-05
305.57691001   6.20984907821e-05   4.39103671728e-05
310.070688099   0.000148785621533   0.000105207569287
314.564466187   0.0   0.0
318.465643453

#S 26 Spectrum ID ('bank1', (3, 127))
#N 3
#L lambda_T(Angstroms)  Intensity(Counts/A)  Sigma(Counts/A)
143.800898828   0.000116441879273   0.000116442178232
148.294676917   0.000180830197377   0.0001140655546
152.788455005   0.000119065225796   7.11275545692e-05
157.282233094   3.88702624618e-05   3.88702910871e-05
161.776011182   2.63040177974e-05   2.63040377752e-05
166.26978927   5.29689244934e-05   5.29689647232e-05
170.763567359   5.11715903127e-06   5.11716626593e-06
175.257345447   5.62588574806e-05   5.62589370196e-05
179.751123535   5.55155592215e-05   5.25958756109e-05
184.244901624   5.89677764687e-05   4.9233043352e-05
188.738679712   2.04732252649e-05   2.04732411262e-05
193.232457801   6.52912406365e-05   4.61679296316e-05
197.726235889   0.000101264535458   6.05078825245e-05
202.220013977   4.41756279644e-05   4.41756571878e-05
206.713792066   9.27317798489e-06   6.55714600574e-06
211.207570154   0.000115521733092   8.16864371586e-05
215.701348243   0.00011814027775   7.84758946313e-05
220.195126331   0.000168632890272   8.49284464157e-05
224.688904419   6.41837601451e-05   3.70565929065e-05
229.182682508   6.47571600103e-05   4.57902780772e-05
233.676460596   0.000130036099938   6.73664311673e-05
238.170238685   0.000161787648158   7.89830060503e-05
242.664016773   0.000266053482639   0.000126051231184
247.157794861   0.000208511918959   0.000120385041241
251.65157295   1.55960029835e-05   9.00440354553e-06
256.145351038   0.00011584189527   8.1912732589e-05
260.639129126   5.39381019516e-05   3.81400641904e-05
265.132907215   0.0   0.0
269.626685303   6.15094435496e-05   4.34938378552e-05
274.120463392   0.00015817649295   0.000101147032183
278.61424148   7.9728475233e-05   7.97286128226e-05
283.108019568   0.0   0.0
287.601797657   7.37783421871e-05   7.37784517596e-05
292.095575745   1.85443018411e-05   1.85443293824e-05
296.589353834   5.24441912823e-05   5.24442287377e-05
301.083131922   8.52918304642e-05   6.03216437513e-05
305.57691001   5.7608402144e-05   5.76084468445e-05
310.070688099   0.0   0.0
314.564466187   1.85253847513e-05   1.06957408092e-05
318.465643453
