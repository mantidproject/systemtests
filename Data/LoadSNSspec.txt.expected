wksp.getNumberHistograms() == 4
wksp.blocksize() == 39
